src/Geralt.Tests/XChaCha20Poly1305Tests.cs

```diff
@@ -183,4 +183,208 @@ public void Decrypt_InvalidKey()
         key = new byte[XChaCha20Poly1305.KeySize + 1];
         Assert.ThrowsException<ArgumentOutOfRangeException>(() => XChaCha20Poly1305.Decrypt(plaintext, Ciphertext, Nonce, key));
     }
+
+    [TestMethod]
+    public void IncrementalEncryptDecrypt_ValidInputs()
+    {
+        Span<byte> header = stackalloc byte[IncrementalXChaCha20Poly1305.HeaderBytes];
+        Span<byte> plaintext = stackalloc byte[Plaintext.Length];
+        Span<byte> ciphertext = stackalloc byte[Plaintext.Length + IncrementalXChaCha20Poly1305.TagSize];
+        byte outTag = 0xFF;
+        using var encryptor = new IncrementalXChaCha20Poly1305(false, header, Key);

```

Comment:
Please can you add `decryption: false` and `decryption: true` everywhere.

Reply:
I missed this.  Adding it now.